test/unit/specs/util/flssUtil.spec.js

```diff
@@ -0,0 +1,30 @@
+import { fetchData } from '@/util/flssUtils';
+import flssLoanQuery from '@/graphql/query/flssLoansQuery.graphql';
+
+describe('flssUtils.js', () => {
+	describe('fetchData', () => {
```

Comment:
This is a great test!

I think you should also test the result returned by the function, which you can do with a couple additions to this test.
First, create a separate object to expect the fetchData function to return and add it to dataObj:
```
const result = { values: [], totalCount: 0 };
const dataObj = {
	data: { fundraisingLoans: result }
};
```

And then you can create another test that just checks for the result:
```
it('Returns the currently fundraising loans', done => {
	fetchData(loanQueryFilters, apollo).then(data => {
		expect(data).toBe(result);
		done();
	});
});
```

Reply:
great suggestions! will do!